project/AppDependencies.scala

```diff
@@ -3,12 +3,12 @@ import sbt._
 object AppDependencies {
 
   private val bootstrapVersion = "9.0.0"
```

Comment:
I believe the most recent bootstrap version as of this change is 9.1.0

Reply:
⚡